pkgs/top-level/python-packages.nix

```
@@ -936,6 +938,8 @@ in {
 
   maxminddb = callPackage ../development/python-modules/maxminddb { };
 
+  mediafile = disabledIf (!isPy3k) (callPackage ../development/python-modules/mediafile { });
```

move disable logic into the expression (this is specific to `buildPythonX`)
```suggestion
  mediafile = callPackage ../development/python-modules/mediafile { };
```